src/Microsoft.ApiDesignGuidelines.Analyzers/Core/PropertiesShouldNotBeWriteOnly.cs

```diff
@@ -6,20 +6,19 @@
 using Analyzer.Utilities;
 
 namespace Microsoft.ApiDesignGuidelines.Analyzers
-{                   
+{
     /// <summary>
     /// CA1044: Properties should not be write only
     /// </summary>
     public abstract class PropertiesShouldNotBeWriteOnlyAnalyzer : DiagnosticAnalyzer
```

Comment:
You can make this class `sealed` and delete the CSharp and VB derived classes since this analyzer is language-agnostic.